src/main/java/org/fogbowcloud/app/core/datastore/managers/JobDBManager.java

```diff
@@ -0,0 +1,48 @@
+package org.fogbowcloud.app.core.datastore.managers;
+
+import org.fogbowcloud.app.core.datastore.repositories.JobRepository;
+import org.fogbowcloud.app.core.models.job.Job;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+public class JobDBManager {
+    private static JobDBManager instance;
+
+    @Autowired
```

Comment:
Is this notation required even using the set method?

Reply:
Fixed in 8f304327bf8689f7bd7038b2332833c69e82f3bc.